src/fmu/dataio/dataio.py

```diff
@@ -1024,12 +1022,6 @@ def export(
                 self._metafile, self._metadata, savefmt=self.meta_format
             )
             logger.info("METAFILE %s", self._metafile)
-        else:
-            warn(
-                "The metadatafile exists already. use 'force' or delete the "
-                "current case folder if a new metadata are requested.",
-                UserWarning,
-            )
```

Comment:
This was redundant as the same warning will be issued from `generate_case_metadata` which is called in the if